CHANGELOG.md

```diff
@@ -9,6 +9,8 @@ and this project adheres to [Semantic Versioning](http://semver.org/)
 
 ### Added 🚀
 
+-   Highlight label while hovering over building, draw a temporary label for hovered buildings that have none ([#1529](https://github.com/MaibornWolff/codecharta/issues/1529))
```

Comment:
Could you recommend a tool?
When using windows snipping tool the mouse is not visible in the resulting picture.

Reply:
I sadly do not know what tools to use with Windows. I am not certain but there might be a keyboard shortcut that could be used with the built-in snipping tool. Otherwise ignore my comment :)

Reply:
Seems like showing the cursor is indeed not supported by Snipping Tool, I get a better software for next time